epochStart/metachain/systemSCs.go

```diff
@@ -885,4 +929,7 @@ func (s *systemSCProcessor) EpochConfirmed(epoch uint32) {
 		"epoch", epoch,
 		"maxNodes", s.maxNodes,
 	)
+
+	s.flagGovernanceV2Enabled.Toggle(epoch == s.governanceV2EnableEpoch)
```

Comment:
governanceEnable - simple :) no need for V2 here

Reply:
done